packages/cli/src/prompts.ts

```diff
@@ -144,6 +144,8 @@ The steps are: I. Fetching configs, II. Calculating difference and III. Applying
 
   public static readonly CANCELED = 'Canceling...'
   public static readonly CREDENTIALS_HEADER = (serviceName: string): string => `Please enter your ${serviceName} credentials:`
+  public static readonly GO_TO_BROWSER = (url: string): string => `Please follow the steps from authenticating in your browser. If a new tab has not opened automatically, please can go to ${url}`
```

Comment:
```suggestion
  public static readonly GO_TO_BROWSER = (url: string): string => `Please follow the steps for authenticating in your browser. If a new tab has not opened automatically, please go to ${url}`
```
not sure about the copy, just fixed the mistakes, feel free to change the message if you want to